src/app/pages/user/dialogs/base/user-details-form/user-details-form.scss

```diff
@@ -0,0 +1,155 @@
+@import '../../../../../app.component.scss';
+
+* {
+  direction: rtl;
+  text-align: right;
+}
+
+mat-dialog-actions {
+  display: flex;
+  justify-content: space-between;
+}
+
+mat-dialog-actions button {
+  text-align: center;
+}
+
+.mat-dialog-actions {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  text-align: center;
+}
+
+.mat-dialog-actions button {
+  text-align: center;
+}
+
+.mat-form-field {
+  //  background-color: aqua;
```

Comment:
throughout this file, can you please remove commented code?

Reply:
Done